index.js

```diff
@@ -33,6 +34,7 @@ let bot = controller.spawn({
 bot.startRTM((err, bot, payload) => {
   if (err) { throw new Error('Could not connect to Slack') }
   debug('Estamos coneptados al Eslá')
+  giveInvites(bot)
```

Comment:
I don't know how I feel about calling the cron function here, let me understand it a little more and i'll give you my thoughts.

Reply:
Actually this was the best place that I found to call it for multiple reasons:
+ We can be sure that Leonel is connected to Slack.
+ We can start a conversation, passing as an argument the `bot`.
+ If Leonel doesn't post on the `Anuncios` channel the first day of the month, we know that the connection with the server is having some issues; then we might run the script manually because the cron job only runs the first day of each month.